prow/line/line.go

```diff
@@ -334,7 +334,7 @@ func deleteKubeJob(k deleteClient, job kube.Job) error {
 	return nil
 }
 
-func SetJobStatus(k *kube.Client, podName, jobName, state, desc, url string) error {
+func SetJobStatus(k *kube.Client, podName, jobName, state, desc, url, agent string) error {
```

Comment:
Deck doesn't start the job, though. We can change the `line.Start*Job` functions to accept the agent string and set it once.